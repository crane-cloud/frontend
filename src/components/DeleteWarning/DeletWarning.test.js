import React from "react";
import { render } from "@testing-library/react";

import DeletWarning from ".";

describe("Test the delete warning component", () => {
  const AlignedDeletWarningComponent = render(
    <DeletWarning textAlignment="center" />
  );
  const UnalignedDeletWarningComponent = render(
    <DeletWarning textAlignment={false} />
  );

  it("checks if the delete warning component matches the snapshot", () => {
    expect(AlignedDeletWarningComponent).toMatchSnapshot();
  });
});
