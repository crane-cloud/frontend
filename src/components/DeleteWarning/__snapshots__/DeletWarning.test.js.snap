// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test the delete warning component checks if the delete warning component matches the snapshot 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="DeleteWarning center"
      >
        <small>
          Note that this action is irreversible.
        </small>
      </div>
    </div>
    <div>
      <div
        class="DeleteWarning"
      >
        <small>
          Note that this action is irreversible.
        </small>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="DeleteWarning center"
    >
      <small>
        Note that this action is irreversible.
      </small>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
